### ISSUE TITLE
Home Page

### ISSUE THREAD

**Issue by Papaya-Voldemort** [skbidisigma1/stocks-sim#1]
So the home page looks great except for the fact that the footer is on the side (I think it would look better on the bottom) and the title is very close to the edge, moving the footer down below (Possibly a dark colored footer) may be able to improve design quality!

**Comment by skbidisigma1** [comment]
Yeah I designed the homepage on mobile so it didn’t exactly turn out as planned on desktop. I’ll get on a pc and work on it more, should look a lot better. It could probably use a lot of changes so I’ll work on it

**Comment by Papaya-Voldemort** [comment]
Ok sounds good, I can try it too, but I am not very fluent in CSS which I think will need to be changed to adapt it, I think a Flexbox would help but I can't be sure (https://www.w3schools.com/css/css3_flexbox.asp) I will work with some AI to try to do it.

**Comment by skbidisigma1** [comment]
Yeah that's totally fine, I'm still very new to html/css so I've been getting a lot of help as well. I'll look into that

**Comment by skbidisigma1** [comment]
So I looked at the code again and apparently there’s no footer set? I guess I just forgot to commit that change then

**Comment by Papaya-Voldemort** [comment]
Ok so I tried to fix it, hope I did not undo your stuff, and it worked... Sort of... I have to do my chores now so I may not be back on.

**Comment by skbidisigma1** [comment]
Yeah looks great nice job!

**Comment by Papaya-Voldemort** [comment]
But I think I ruined the other pages 💀 I am so sorry...

**Comment by skbidisigma1** [comment]
Nah all good! I'm pretty sure the issue is just the header not rendering right, should be an easy fix. And never worry much about breaking or undoing things I'm pretty sure we can just undo the commit if needed or just fix it

**Comment by Papaya-Voldemort** [comment]
I just found we can make a wiki... I guess I know what the next 20 hours of my life will be spent doing...

**Comment by skbidisigma1** [comment]
Yeah lol idk who they get to make wikis but it must take forever
